main.py

```diff
@@ -0,0 +1,51 @@
+from flask import Flask, render_template, request, redirect, url_for
+import pandas as pd
+import json
+import plotly
+import plotly.express as px
+from plotly.subplots import make_subplots
+import plotly.graph_objects as go
+ 
+app = Flask(__name__)
+
+df = pd.read_csv('women_EU2.csv', sep=',')
+
+@app.route('/', methods=['GET','POST'])
+def search():
```

Comment:
What if you split this route into two - `'/'` which would be `index` with `'GET'` and `'/search'` with `'POST'`? 😏